### PR TITLE
Stats/Stackdriver: Add user agent as client header

### LINKED ISSUE #194: Exporter/Stats/Stackdriver: Add user agent as client header
Equivalent to https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/blob/master/stats.go#L80-L81.

### PR DESCRIPTION
**Pull request by mayurkale22**
Fixes #194

### CODE REVIEW

**Comment by mayurkale22** [comment]
I had offline discussion with @JustinBeckwith, decided to add options at ```create``` method level, as  currently it is not supported (bug) with top level option (https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-stackdriver/src/stackdriver-monitoring.ts#L23).

File: packages/opencensus-exporter-stackdriver/src/stackdriver-monitoring.ts [hunk 1]
```diff
@@ -151,10 +160,13 @@ export class StackdriverStatsExporter implements StatsEventListener {
...
-        monitoring.projects.timeSeries.create(request, (err: Error) => {
-          this.logger.debug('sent time series', request.resource.timeSeries);
-          err ? reject(err) : resolve();
-        });
+        monitoring.projects.timeSeries.create(
+            request, {headers: OC_HEADER, userAgentDirectives: [OC_USER_AGENT]},
+            (err: Error) => {
+              this.logger.debug(
+                  'sent time series', request.resource.timeSeries);
+              err ? reject(err) : resolve();
```
[the remark this answers is not in the document]
Comment (mayurkale22) [reply packages/opencensus-exporter-stackdriver/src/stackdriver-monitoring.ts]: Done.

File: packages/opencensus-exporter-stackdriver/src/stackdriver-monitoring.ts [hunk 2]
```diff
@@ -173,10 +185,12 @@ export class StackdriverStatsExporter implements StatsEventListener {
       };
 
       return new Promise((resolve, reject) => {
-        monitoring.projects.metricDescriptors.create(request, (err: Error) => {
-          this.logger.debug('sent metric descriptor', request.resource);
-          err ? reject(err) : resolve();
-        });
+        monitoring.projects.metricDescriptors.create(
+            request, {headers: OC_HEADER, userAgentDirectives: [OC_USER_AGENT]},
```
[the remark this answers is not in the document]
Comment (mayurkale22) [reply packages/opencensus-exporter-stackdriver/src/stackdriver-monitoring.ts]: As per my understanding, User Agent requirement is only Stackdriver stats exporter specific. For Stackdriver Trace exporter, we add agent information as a part of the label (example: ```g.co/agent -> opencensus-node [{version}]```). This is implemented in all the libraries except node. Issue https://github.com/census-instrumentation/opencensus-node/issues/39 -> to address that.

[the remark this answers is not in the document]
Comment (draffensperger) [reply packages/opencensus-exporter-stackdriver/src/stackdriver-monitoring.ts]: Ah, makes sense, then keeping these constants in this file only seems fine.

**Pull request merged**